advanced/Scripts/update.sh

```diff
@@ -62,7 +66,7 @@ GitCheckUpdateAvail() {
   fi
 
   # Change back to original directory
-  cd "${curdir}"
+  cd "${curdir}" || exit
```

Comment:
Unless I'm mistaken in its purpose, if `cd` fails, we'll want to use `exit 1` to signify that it quit with an error.